app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -159,24 +159,36 @@ public void askReasonAndExecute(Media media,
-        String[] reasonListEnglish = {"Eng1", "Eng2", "Eng3"};
+        final String[] reasonList;
+        final String[] reasonListEnglish;
 
         if (problem == ReviewController.DeleteReason.SPAM) {
-            reasonList[0] = context.getString(R.string.delete_helper_ask_spam_selfie);
-            reasonList[1] = context.getString(R.string.delete_helper_ask_spam_blurry);
-            reasonList[2] = context.getString(R.string.delete_helper_ask_spam_nonsense);
-            reasonListEnglish[0] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_selfie);
-            reasonListEnglish[1] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_blurry);
-            reasonListEnglish[2] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_nonsense);
+            reasonList = new String[] {
+                context.getString(R.string.delete_helper_ask_spam_selfie),
+                context.getString(R.string.delete_helper_ask_spam_blurry),
+                context.getString(R.string.delete_helper_ask_spam_nonsense)
+            };
+            reasonListEnglish = new String[] {
+                getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_selfie),
+                getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_blurry),
+                getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_spam_nonsense)
+            };
         } else if (problem == ReviewController.DeleteReason.COPYRIGHT_VIOLATION) {
-            reasonList[0] = context.getString(R.string.delete_helper_ask_reason_copyright_press_photo);
-            reasonList[1] = context.getString(R.string.delete_helper_ask_reason_copyright_internet_photo);
-            reasonList[2] = context.getString(R.string.delete_helper_ask_reason_copyright_logo);
-            reasonListEnglish[0] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_press_photo);
-            reasonListEnglish[1] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_internet_photo);
-            reasonListEnglish[2] = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_reason_copyright_logo);
+            reasonList = new String[] {
+                context.getString(R.string.delete_helper_ask_reason_copyright_press_photo),
+                context.getString(R.string.delete_helper_ask_reason_copyright_internet_photo),
+                context.getString(R.string.delete_helper_ask_reason_copyright_logo),
+                context.getString(R.string.delete_helper_ask_reason_copyright_no_freedom_of_panorama)
+            };
+            reasonListEnglish = new String[] {
```

Comment:
Avoid lines longer than 80 characters. Give a line break to all the long lines.

Otherwise, PR looks good to me.

Reply:
That's 102 characters, it is not that bad I would say, actually a vast improvement over what existed before :-)
In that case I think keeping the 102 characters is more readable that inserting break, so I would say it is OK.